src/lib/recurringPurposes.js

```diff
@@ -1,18 +1,248 @@
-// @ts-check
```

Comment:
Why this removal?

Reply:
I had plenty of type errors when I opened the unmodified file, so I assumed it wasn't really used